app/src/main/java/com/neiapp/spocan/Models/User.java

```diff
@@ -8,10 +9,31 @@
 public class User {
     private String nickname;
     private UserType type;
+    private int amount_inititatives;
+    private boolean isAdmin;
```

Comment:
mismo comentario respecto del backend. Por lo general la convención es que los booleanos sean sustantivos y luego su getter les agrega el is. Por ejemplo

**private boolean admin;

public boolean isAdmin() { return this.admin }**